docs/src/content/docs/de/reference/configuration.mdx

```diff
@@ -398,7 +527,7 @@ Eine Seite kann diese Einstellung oder den Linktext und/oder die URL mit Hilfe d
 ### `favicon`
 
 **Typ:** `string`  
-**Standard:** `'/favicon.svg'`
+**Voreinstellung:** `'/favicon.svg'`
```

Comment:
I cannot suggest a change for it as it's outside of the review range but the second example in this section is missing the change from https://github.com/withastro/starlight/pull/1627/files#diff-889e6ae5a36c15d03a30012e2936df94a63753b335bd5a8c56fac9a42e5c1dff